container/features/src/main/resources/features.xml

```diff
@@ -805,7 +809,8 @@
         <feature>commons-cli</feature>
         <feature>opennms-collection-api</feature>
         <feature>opennms-provisioning-api</feature>
-        <bundle dependency="true">wrap:mvn:org.jinterop/j-interop/${jinteropVersion}</bundle>
+        <feature>jcifs</feature>
+        <bundle dependency="true">wrap:mvn:org.jinterop/j-interop/${jinteropVersion}$Import-Package=jcifs,jcifs.smb,jcifs.util,jcifs.netbios,ndr,rpc,rpc.core,rpc.ncacn_np,rpc.pdu,rpc.security.ntlm</bundle>
```

Comment:
Update these bundles with `com.github.skyghis/j-interop-ng`  and `com.github.skyghis/j-interop-ng-deps`